react/CategoryMenu.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'react-apollo'
+
+import getCategories from './queries/categoriesQuery.gql'
+import CategoryItem from './CategoryItem'
+
+/**
+ * Component that represents the menu containing the categories of the store
+ */
+class CategoryMenu extends Component {
+  render() {
+    const { data: { categories } } = this.props
```

Comment:
Same comment about the `loading` status.
[Example](https://github.com/vtex-apps/dreamstore/blob/master/react/Shelf.js#L50).